easybuild/easyconfigs/p/PyTorch-extensions/PyTorch-extensions-2019.01-foss-2019a-Python-3.7.2.eb

```diff
@@ -0,0 +1,55 @@
+easyblock = 'PythonBundle'
+
+name = 'PyTorch-extensions'
+version = '2019.01'
```

Comment:
additional benefit: we don't need to come up with our own versioning scheme, we can just version the bundle using the `PyTorch-Geometric` version